projects/client-api-react/src/stories/GraphistryJS.stories.jsx

```diff
@@ -88,6 +95,108 @@ export const InstantiateGraphistryJS = (args) => {
     );
 }
 
+const SUBSCRIPTION_TEST = [
```

Comment:
this test is useful but maybe not for standard storybook docs... by default don't export, and we only export in dev via manual code change? or make a separate stories file for testing?